panel/pane/alert.py

```diff
@@ -0,0 +1,55 @@
+"""Bootstrap inspired Alerts
+
+See https://getbootstrap.com/docs/4.0/components/alerts/
+"""
+import param
+
+from panel.pane.markup import Markdown
+
+ALERT_TYPES = ["primary", "secondary", "success", "danger", "warning", "info", "light", "dark"]
+
+
+class Alert(Markdown):
+    """
+    An Alert that renders Markdown
+
+    - CSS Styling is done via the classes `alert` and `alert-TYPE`, where TYPE is the alert_type.
+    - sizing_mode is set to `stretch_width` by default
+    """
+
+    alert_type = param.ObjectSelector("primary", objects=ALERT_TYPES)
+
+    def __init__(
+        self, text: str, **kwargs,
+    ):

```

Comment:
Sticking with this style for now:

```suggestion
    def __init__(self, text: str, **kwargs):
```